modules/gapi/include/opencv2/gapi/streaming/onevpl/accel_types.hpp

```diff
@@ -65,6 +65,10 @@ GAPI_EXPORTS Device create_dx11_device(Device::Ptr device_ptr,
                                        const std::string& device_name);
 GAPI_EXPORTS Context create_dx11_context(Context::Ptr ctx_ptr);
 
+GAPI_EXPORTS Device create_vaapi_device(Device::Ptr device_ptr,
+                                       const std::string& device_name,
+                                       int file_description = -1);
```

Comment:
It seems one more space is required for the proper alignment?

Reply:
fixed